### PR TITLE
[1LP][RFR]test_sdn_port_net_prov for azure is blocked by BZ

### PR DESCRIPTION
**Pull request by mmojzis**

### CODE REVIEW

File: cfme/tests/networks/test_sdn_ports.py [hunk 1]
```diff
@@ -44,6 +45,7 @@ def test_sdn_port_detail_name(provider, appliance):
             pass
 
 
+@pytest.mark.meta(blockers=[BZ(1718495, forced_streams=['5.11'])])
```
Comment (john-dupuy) [inline cfme/tests/networks/test_sdn_ports.py]: @mmojzis I see that this BZ is verified in CFME 5.11, why is `forced_streams` necessary here?

Comment (mmojzis) [reply cfme/tests/networks/test_sdn_ports.py]: @john-dupuy Thanks for pointing that out. I tested it with already verified BZ and then forgot to change to correct BZ. It should be okay now.

File: cfme/tests/networks/test_sdn_ports.py [hunk 2]
```diff
@@ -44,6 +45,7 @@ def test_sdn_port_detail_name(provider, appliance):
             pass
 
 
+@pytest.mark.meta(blockers=[BZ(1758593, forced_streams=['5.11'])])
```
Comment (jawatts) [inline cfme/tests/networks/test_sdn_ports.py]: We do not need the force stream here:
```
In[4]: appliance.version
Out[4]: Version('5.11.0.27')
In[5]: from cfme.utils.blockers import BZ
In[6]: BZ(1758593).blocks
Out[6]: True
```

Comment (mmojzis) [reply cfme/tests/networks/test_sdn_ports.py]: Forced stream removed.

**Comment by mmojzis** [comment]
> One small change but otherwise it looks good

@jawatts Changed.

**Pull request merged**